recipes/imagemagick/all/test_package/test_package.cpp

```diff
@@ -0,0 +1,23 @@
+#include <iostream>
+#include <fstream>
+#include <MagickCore/MagickCore.h>
+
+int main()
+{
+    size_t version, range, depth;
```

Comment:
If possible, it would be cool if you converted this c++ source to a pure c example.
This would test whether, if some dependency uses c++, the c++ runtime is added correctly.